packages/kit/src/views/Staking/pages/ProtocolDetailsV2/index.tsx

```
@@ -754,11 +794,19 @@
         )}
       />
       <Page.Body pb="$5">
-        {/* {isEventActive ? (
+        {result?.countDownAlert &&
+        result.countDownAlert.startTime &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)